sormas-ui/src/main/java/de/symeda/sormas/ui/importer/DataImporter.java

```diff
@@ -157,7 +157,8 @@ public DataImporter(File inputFile, boolean hasEntityClassRow, UserDto currentUs
 	/**
 	 * Opens a progress layout and runs the import logic in a separate thread.
 	 */
-	public void startImport(Consumer<StreamResource> errorReportConsumer, UI currentUI, boolean duplicatesPossible)
+	public void startImport(Consumer<StreamResource> errorReportConsumer, UI currentUI, boolean duplicatesPossible,
```

Comment:
The `singleColumnImport` should be required on the constructor and add an overload to avoid changing all the import code.

And reformat the code